apps/mobile/src/features/Send/useCases/ListAmountsToSend/EditAmount/EditAmountScreen.tsx

```
@@ -51,15 +51,24 @@ export const EditAmountScreen = () => {
   } = useTransfer()
 
   const amount = targets[selectedTargetIndex].entry.amounts[selectedTokenId]
+
+  if (!amount) {
+    // If amount is not found, navigate back to the token selection screen
+    React.useEffect(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)